<?xml version="1.0"?>
<launch>

    <arg name="sim" default="false" doc="Disables checks for hardware, since we are in simulation."/>
    <arg name="minibot" default="false"/>
    <arg name="wolfgang" default="true"/>
    <arg name="use_game_settings" default="false"/>

    <param name="/simulation_active" value="$(arg sim)" />
    
    <group if="$(arg wolfgang)">
        <rosparam file="$(find bitbots_hcm)/config/hcm_wolfgang.yaml" command="load"/>
            <group if="$(arg use_game_settings)">
                <rosparam command="load" file="$(find bitbots_bringup)/config/game_settings.yaml" />
            </group>
        <node name="hcm" pkg="bitbots_hcm" type="hardware_control_manager.py" args="" output="screen" >
        </node>
    </group>

    <node name="pause" pkg="bitbots_hcm" type="pause.py" />

</launch>
